tests/docker_service_fixtures.py

```diff
@@ -258,3 +259,26 @@ def spanner_responsive(host: str) -> bool:
 async def spanner_service(docker_services: DockerServiceRegistry) -> None:
     os.environ["SPANNER_EMULATOR_HOST"] = "localhost:9010"
     await docker_services.start("spanner", timeout=60, check=spanner_responsive)
+
+
+async def mssql_responsive(host: str) -> bool:
+    try:
+        port = 11433
+        user = "app"
+        database = "db"
+        password = "super-secret"
+        conn = await pyodbc.connect(
+            connstring=f"driver={{ODBC Driver 18 for SQL Server}};server={host},{port}; database={database}; UID={user};",
```

Comment:
Is this meant to have double brackets instead of the one? {{ }} 
```suggestion
            connstring=f"driver={ODBC Driver 18 for SQL Server};server={host},{port}; database={database}; UID={user};",
```

Reply:
Oh, f-strings + the literal curly